ludwig/features/binary_feature.py

```diff
@@ -400,11 +405,10 @@ def populate_defaults(output_feature):
                 'threshold': 0.5,
                 'robust_lambda': 0,
                 'confidence_penalty': 0,
+                'pos_ce_weight': 1,
                 'weight': 1
             }
         )
-        set_default_value(output_feature, 'threshold', 0.5)
         set_default_value(output_feature, 'dependencies', [])
-        set_default_value(output_feature, 'weight', 1)
```

Comment:
This parameter is needed for balacing different losses in multi task learning, don't remove it.